src/Config_Command.php

```diff
@@ -224,6 +230,14 @@ public function get( $_, $assoc_args ) {
 		$wp_config_vars      = self::get_wp_config_vars( get_defined_vars(), $wp_cli_original_defined_vars, 'variable', array( 'wp_cli_original_defined_vars' ) );
 		$wp_config_constants = self::get_wp_config_vars( get_defined_constants(), $wp_cli_original_defined_constants, 'constant' );
 
+		$get_constant        = ! empty( $assoc_args['constant'] );
+		$get_global          = ! empty( $assoc_args['global'] );
+
+		if( $get_constant || $get_global ){
+			return $this->return_constant_or_global( $assoc_args, $get_constant, $wp_config_constants, $wp_config_vars );
```

Comment:
This should be `WP_CLI::log()` to display the value, and then `return` to end method execution.